RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -0,0 +1,183 @@
+#ifndef RecoTracker_LSTCore_interface_alpaka_Constants_h
+#define RecoTracker_LSTCore_interface_alpaka_Constants_h
+
+#include <alpaka/alpaka.hpp>
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+#ifdef CACHE_ALLOC
+#include "HeterogeneousCore/AlpakaInterface/interface/CachedBufAlloc.h"
+#endif
+
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
+#include <cuda_fp16.h>
+#endif
+
+namespace SDL {
+// Half precision wrapper functions.
+#if defined(FP16_Base)
+#define __F2H __float2half
+#define __H2F __half2float
+  typedef __half float FPX;
+#else
+#define __F2H
+#define __H2F
+  typedef float FPX;
+#endif
+
+  using Idx = alpaka_common::Idx;
+  using Dim = alpaka_common::Dim3D;
+  using Dim1d = alpaka_common::Dim1D;
+  using Vec = alpaka_common::Vec3D;
+  using Vec1d = alpaka_common::Vec1D;
+  using WorkDiv = alpaka_common::WorkDiv3D;
+
+  using Acc = ALPAKA_ACCELERATOR_NAMESPACE::Acc3D;
+  using Dev = ALPAKA_ACCELERATOR_NAMESPACE::Device;
+  using DevHost = ALPAKA_ACCELERATOR_NAMESPACE::DevHost;
+  using QueueAcc = ALPAKA_ACCELERATOR_NAMESPACE::Queue;
```

Comment:
Please use the same naming as in `HeterogeneousCore/AlpakaInterface/interface/config.h` to avoid confusion with other packages:
```suggestion
  using Acc3D = ALPAKA_ACCELERATOR_NAMESPACE::Acc3D;
  using Device = ALPAKA_ACCELERATOR_NAMESPACE::Device;
  using DevHost = ALPAKA_ACCELERATOR_NAMESPACE::DevHost;
  using Queue = ALPAKA_ACCELERATOR_NAMESPACE::Queue;
```